server/index.js

```diff
@@ -1,7 +1,7 @@
-import express from 'express';
-import router from './routes';
+const express = require('express');
+const router = require('./routes');
 
-import json from 'body-parser';
+const json = require('body-parser');
```

Comment:
or just use express.json()

Reply:
was going to change this in a different branch where i'm just cleaning up the server a bit